INSTALL.md

```diff
@@ -11,16 +11,21 @@ following sections describe them for the supported platforms.
 * GNU Automake
 * GNU Libtool
 * C compiler
-* C Library Development Libraries and Header Files
+
+### C Library Development Libraries and Header Files
 * pkg-config
 * uriparser
 * libgcrypt
 
 The following are dependencies only required when building test suites.
 * Integration test suite (see ./configure option --with-simulatorbin):
-** OpenSSL
+
+   OpenSSL libraries and header files
```

Comment:
Same as above, but not a section this time just indentation?

Reply:
Same as above, but not a section this time just indentation?

The “**” markup for nested indentation is not supported—at least for the GitHub flavor of markup language.   So you get this confusing display:

* Integration test suite (see ./configure option --with-simulatorbin): ** OpenSSL

Take a look yourself at
https://github.com/tpm2-software/tpm2-tss/blob/master/INSTALL.md

Reply:
This formatting problem was just fixed a week later with pull #791:
Fix broken formatting / bullet for OpenSSL dependency.